run

```diff
@@ -666,57 +666,65 @@ build() {
       fi
   done
 
-  echo "Building Holohub"
-
-  # Application to build
+  # Application or operator to build
   if [ "$1" == "" ]; then
-    print_error "Please specify which application to build."
-    echo "You can run ./run list for a full list of applications."
+    print_error "Please specify which operator or application to build."
+    echo "You can run ./run list for a full list of options."
+    exit 1
+  fi
+  project=$(basename "$1")
+  opt_prefix=""
+  if list_apps | grep -Fxq "$project"; then
+    opt_prefix="-D APP_"
+  elif list_operators | grep -Fxq "$project"; then
+    opt_prefix="-D OP_"
```

Comment:
Looks like the only app-specific logic in this block is for benchmarking, good find. Would be good to at least warn if benchmarking is enabled and an operator is requested.

Reply:
https://github.com/nvidia-holoscan/holohub/compare/eb33946194f501a6b39f450d3f5e556733488b25..ba23d2711b64a4acf766f93e98cfe489f55a06bd